docs/content/en/docs/contribution-guidelines/piped-development.md

```diff
@@ -25,3 +25,11 @@ bazelisk run --run_under="cd $PWD && " //cmd/piped:piped -- piped \
 --tools-dir=/tmp/piped-bin \
 --config-file=piped-config.yaml
 ```
+
+## How to load piped image into the local Docker client
```

Comment:
Should it be: "How to run it locally as docker container" 🤔 
And add the `docker run bazel/cmd/piped:image -- help` command as the sample for how it could be used as a locally docker container below this `make load-piped-image` command explanation 👀

Reply:
Sure. Let me fix.

Reply:
Fixed it.